src/pages/reportSighting/ReportForm.jsx

```diff
@@ -314,6 +335,22 @@ export default function ReportForm({
                   sightings: [report],
                 };
 
+                const grecaptchaReady = new Promise(resolve => {
```

Comment:
I think `if (recaptchaPublicKey)` needs to be above this line because if we don't include `recaptcha/api.js`, `window.grecaptcha` won't be defined